torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -185,6 +171,15 @@ std::unordered_set<Node*> AliasDb::getWriters(const Node* n) const {
       }
     }
   }
+
+  // A write to the wildcard set should be considered a write to `n`
+  if (aliasToWrites_.count(AliasInfo::wildcardSet())) {
```

Comment:
This looks right but I don't think is currently being tested

Reply:
this gets indirectly tested by the test above; if we extract something from a list and write to it, it is a write to the wildcard set.

Reply:
So is containedType not being used ? might be safer to test with a python function than list index.